### PR TITLE
add util function to get all the annotations in a single dictionary

### PR DESCRIPTION
**Pull request by almazan**

### CODE REVIEW

File: darwin/torch/utils.py [hunk 1]
```diff
@@ -16,8 +23,7 @@ def load_pil_image(path: Path):
     """
     Loads a PIL image and converts it into RGB.
 
-    Parameters
-    ----------
+    Parameters ----------
```
Comment (Renthal) [inline darwin/torch/utils.py]:
```suggestion
    Parameters 
    ----------
```

File: darwin/torch/utils.py [hunk 2]
```diff
@@ -111,3 +117,83 @@ def polygon_area(x: np.ndarray, y: np.ndarray) -> float:
...
+    if classes[0] == "__background__":
+        classes = classes[1:]
+    extensions = [".jpg", ".jpeg", ".png"]
+    stems = (e.strip() for e in split.open())
+    images_path = []
+    annotations_path = []
+
+    for stem in stems:
+        annotation_path = root / f"annotations/{stem}.json"
+        images = [image for image in root.glob(f"images/{stem}.*") if image.suffix in extensions]
```
Comment (Renthal) [inline darwin/torch/utils.py]:
```suggestion
        images = [image for image in root.glob(f"images/{stem}.*") if image.suffix.lower() in extensions]
```

File: darwin/torch/utils.py [hunk 3]
```diff
@@ -111,3 +117,83 @@ def polygon_area(x: np.ndarray, y: np.ndarray) -> float:
     for x and y coordinates.
     """
     return 0.5 * np.abs(np.dot(x, np.roll(y, 1)) - np.dot(y, np.roll(x, 1)))
+
+
+def get_annotation_dicts(root, split, classes_file):
```
Comment (Renthal) [inline darwin/torch/utils.py]: Docstring is missing :)

Comment (almazan) [reply darwin/torch/utils.py]: Added

**Pull request merged**